front/app/modules/commercial/customizable_navbar/admin/containers/EditPageForm/index.tsx

```diff
@@ -10,16 +10,20 @@ import { isPageInNavbar } from './utils';
 
 // components
 import EditPageFormNavbar from './EditPageFormNavbar';
-import EditPageFormNotInNavbar from './EditPageFormNotInNavbar';
+import EditPageFormNotInNavbar from 'containers/Admin/pagesAndMenu/containers/EditPageForm/EditPageFormNotInNavbar';
+import useFeatureFlag from 'hooks/useFeatureFlag';
 
 const EditPageForm = ({ params: { pageId } }: WithRouterProps) => {
   const navbarItems = useNavbarItems();
+  const customizableNavbarEnabled = useFeatureFlag({
```

Comment:
Since we're already in the `customizable_navbar` module, won't this always be true? I don't actually know either way, but I would guess you can only see this code if your site has `customizable_navbar` enabled, in which case it would always be true

Reply:
> won't this always be true?

No, it just means the module is loaded, and this code is executed (by react-router in this case). But not sure it's a good way to check it here.